modules/compiler/vecz/source/debugging.cpp

```diff
@@ -26,31 +26,39 @@ namespace vecz {
 ///
 /// @param[in] V The value (can be `nullptr`) to be included in the remark
 /// @param[in] Msg The main remark message
+/// @param[in] Note An optional additional note to provide more context/info.
 /// @return The remark message as it is to be printed
-static std::string createRemarkMessage(const Value *V, StringRef Msg) {
+static std::string createRemarkMessage(const Value *V, StringRef Msg,
+                                       StringRef Note = "") {
   std::string helper_str("Vecz: ");
   raw_string_ostream helper_stream(helper_str);
   helper_stream << Msg;
   if (V) {
     if (isa<Instruction>(V)) {
       // Instructions are already prefixed by two spaces when printed
-      V->print(helper_stream, true);
+      V->print(helper_stream, /*IsForDebug*/ true);
```

Comment:
The style used in [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#comment-formatting) is `/*IsForDebug=*/true`, which we use in a few places already as well. Do we want to just stick with that?

Reply:
Huh, I hadn't noticed that. Thanks, I've updated to use the proper style.